gulpfile.js

```diff
@@ -1157,6 +1169,66 @@ gulp.task('importl10n', function(done) {
   locales.downloadL10n(L10N_DIR, done);
 });
 
+gulp.task('gh-pages-prepare', ['web-pre'], function () {
+  console.log();
+  console.log('### Creating web site');
+
+  rimraf.sync(GH_PAGES_DIR);
+
+  // 'vinyl' because web/viewer.html needs its BOM.
+  return merge([
+    vinyl.src(GENERIC_DIR + '**/*', {base: GENERIC_DIR, stripBOM: false})
+         .pipe(gulp.dest(GH_PAGES_DIR)),
+    gulp.src([FIREFOX_BUILD_DIR + '*.xpi',
+              FIREFOX_BUILD_DIR + '*.rdf'])
+        .pipe(gulp.dest(GH_PAGES_DIR + EXTENSION_SRC_DIR + 'firefox/')),
+    gulp.src([CHROME_BUILD_DIR + '*.crx',
+              FIREFOX_BUILD_DIR + '*.rdf'])
+        .pipe(gulp.dest(GH_PAGES_DIR + EXTENSION_SRC_DIR + 'chromium/')),
+    gulp.src('test/features/**/*', {base: 'test/'})
+        .pipe(gulp.dest(GH_PAGES_DIR)),
+    gulp.src(JSDOC_DIR + '**/*', {base: JSDOC_DIR})
```

Comment:
I was puzzled as to why this folder, https://github.com/mozilla/pdf.js/tree/gh-pages/api/draft, wasn't copied over.

It seems that should add `var JSDOC_BUILD_DIR = BUILD_DIR + 'jsdoc/';` with the other constants at the top of the file, and then use that here instead.
Bonus points if this new constant is also used in https://github.com/mozilla/pdf.js/blob/master/gulpfile.js#L932 as well :-)